bundle-custom.Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use OpenShift golang builder image
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16 AS builder
```

Comment:
@cpmeadors Can you remind where `bundle-custom.Dockerfile` is used ?

Reply:
I saw [this test](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_sandboxed-containers-operator/417/pull-ci-openshift-sandboxed-containers-operator-devel-ci-index-openshift-sandboxed-containers-operator-bundle/1803708952588849152) was failing because of it

Reply:
The test you mention is failing because of `COPY internal internal/` step, not because of the golang builder.
Ideally this should go to some separate PR but it is acceptable to make it a preparatory commit here.

Reply:
> That was left out by previous golang bump (https://github.com/openshift/sandboxed-containers-operator/commit/5accd071a837dc74c789dae0fd0de8f3cb6046a7).
>
> This change is not necessary for this pull request. I'm doing it because I'm touching
> the same file in the next commit.

Excellent commit message ! :+1: